pex/common.py

```diff
@@ -15,9 +15,18 @@
 import threading
 import zipfile
 from collections import defaultdict
+from datetime import datetime
 from uuid import uuid4
 
 
+# We use the start of MS-DOS time, which is what zipfiles use (see section 4.4.6 of
+# https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT). ZipInfo expects a 6-tuple of
+# year-month-day-hour-minute-second, which we use here.
+DETERMINISTIC_DATETIME = datetime(
+  year=1980, month=1, day=1, hour=0, minute=0, second=0, tzinfo=None
+).timetuple()[:6]
```

Comment:
I think you're missing an opportunity here to just know about [:6] down on line 113. In other words, always take a `time.struct_time` in `zip_info_from_file` for the `date_time` parameter. If `None`, use the constant defined here (but unsliced). Then slice the `date_time` just once!

Reply:
Ah okay _now_ I get what you were suggesting all along. Thanks! Good recommendation.